styles/product.scss

```diff
@@ -102,6 +102,16 @@
           height: 100% !important;
           margin: 0;
         }
+
+        @media screen and (max-width: 767px) {
```

Comment:
you can use @include breakpoint('md') mixins for media queries

Reply:
yes but i want to use max-width not min-width

Reply:
you could add it to _mixins.scss file